broker/src/main/fabric8/deployment.yml

```diff
@@ -8,10 +8,10 @@ spec:
           periodSeconds: 5
           timeoutSeconds: 1
           tcpSocket:
-            port: 5672
+            port: 5671
```

Comment:
I was just thinking that maybe this would allow for a more *meaningful* check if the broker is really in a state to process messages (as compared to simply being able to open a TCP connection). But I do not know enough about Artemis to argue for this way or the other. If in Artemis the fact that the listener is running on 5672 is equivalent to *Artemis is ready to process messages* then there doesn't seem to be any real benefit.